example/basic/main.go

```diff
@@ -55,11 +57,9 @@ func main() {
 		tag.Insert(barKey.String("bar1")),
 	)
 
-	gauge := meter.GetFloat64Gauge(
-		ctx,
-		oneMetric,
-		lemonsKey.Int(10),
-	)
+	gauge := oneMetric.Handle(ctx, meter, lemonsKey.Int(10))
+
+	measure := measureTwo.Handle(ctx, meter, lemonsKey.Int(10))
```

Comment:
Similar situation here - the `lemonsKey` is going to be ignored, because is not a part of predefined keys, right?

Reply:
I'll update the RFC, this requires detail.